app/view_models/auction_show_view_model.rb

```diff
@@ -73,7 +73,9 @@ def bid_status_class
   end
 
   def bid_status_partial
-    if auction.type == 'reverse' && (over? || available_and_user_is_winning_bidder?)
+    if reverse_auction_over? ||
```

Comment:
Similarly, I wonder if we should be moved to the Rules objects or if the Rules objects can instantiate their own corresponding presenters/view models?